pkg/config/consumer_chain_test.go

```diff
@@ -45,13 +45,22 @@ func TestConsumerChainValidateNoChainId(t *testing.T) {
 	require.Error(t, err)
 }
 
+func TestConsumerChainValidateNoBaseDenom(t *testing.T) {
+	t.Parallel()
+
+	chain := ConsumerChain{Name: "test", LCDEndpoint: "test", ChainID: "test"}
+	err := chain.Validate()
+	require.Error(t, err)
+}
+
 func TestConsumerChainValidateInvalidDenom(t *testing.T) {
 	t.Parallel()
 
 	chain := ConsumerChain{
 		Name:        "test",
 		LCDEndpoint: "test",
 		ChainID:     "test",
+		BaseDenom:   "denom",
```

Comment:
Actually this test is intentionally does not have these fields set, as it would make it an invalid denom info and therefore the validation would fail.